cmd/k8s-operator/operator.go

```diff
@@ -378,13 +378,15 @@ func runReconcilers(opts reconcilerOpts) {
 
 	epsFilter := handler.EnqueueRequestsFromMapFunc(egressEpsHandler)
 	podsSecretsFilter := handler.EnqueueRequestsFromMapFunc(egressEpsFromEgressPGChildResources(mgr.GetClient(), opts.log, opts.tailscaleNamespace))
+	epsFromExtNSvcFilter := handler.EnqueueRequestsFromMapFunc(epsFromExternalNameService(mgr.GetClient(), opts.log))
 
 	err = builder.
 		ControllerManagedBy(mgr).
 		Named("egress-eps-reconciler").
 		Watches(&discoveryv1.EndpointSlice{}, epsFilter).
 		Watches(&corev1.Pod{}, podsSecretsFilter).
 		Watches(&corev1.Secret{}, podsSecretsFilter).
+		Watches(&corev1.Service{}, epsFromExtNSvcFilter).
```

Comment:
Additionally had to add this watch as the EndpointSlice reconciler checks for this service to have Valid and Configured conditions before proceeding